methods_and_recursion/bottles_of_beer.rb

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
+
+def plural(bottles, words)
+  words += 's' if bottles != 1
+  if bottles.zero?
+    'No more bottles'
+  else
+    "#{bottles} #{words}"
+  end
+end
+
+def no_bottles_left
+  puts 'No more bottles of beer on the wall, no more bottles of beer.'
```

Comment:
So, on line 23, you check to see if bottles > 1, if it's not, i.e. it's the last bottle, you call `no_bottles_left`.  Which means you don't need the if check on line 5...